fastlane/Fastfile

```diff
@@ -541,6 +541,58 @@ lane :beta do
   )
 end
 
+lane :download_app_store_reviews do
+  desc "Download App Store reviews and save them to a file"
+
+  UI.message "Logging into App Store Connect..."
+
+  # Use credentials from the Appfile
+  apple_id = CredentialsManager::AppfileConfig.try_fetch_value( :apple_id )
+  itc_team_id = CredentialsManager::AppfileConfig.try_fetch_value( :itc_team_id )
+
+  # Explicitly login with credentials
+  Spaceship::Tunes.login( apple_id )
+  Spaceship::Tunes.select_team( team_id: itc_team_id, team_name: nil )
+
+  # Select the app using the bundle ID from the Appfile
+  UI.message "Finding app with bundle ID: #{IOS_BUNDLE_ID}"
+  app = Spaceship::Tunes::Application.find( IOS_BUNDLE_ID )
+
+  unless app
+    UI.abort_with_message! "Could not find app with bundle ID: #{IOS_BUNDLE_ID}"
+  end
+
+  # Fetch reviews
+  UI.message "Fetching reviews for #{app.name}..."
+  reviews = app.ratings.reviews
+
+  # Create an array to hold processed reviews
+  processed_reviews = []
+  reviews.each do | review |
+    # Log the review details
+    UI.message "Review ID: #{review}"
+    # Convert review object to a hash with the fields you want
+    processed_review = {
+      rating: review.rating.to_i,
+      title: review.title,
+      review_content: review.review,
+      nickname: review.nickname,
+      country: review.store_front,
+      last_modified: review.last_modified,
+      edited: review.edited,
+      responded: review.responded?,
+      developer_response: review.developer_response.response
```

Comment:
Do we know that there is always a developer_response? Will this fail if there's not a developer response?

Reply:
No, there is not always a response, and in cases there is none it does not fail but save null to the JSON.